### DISCUSSION TITLE
[Feature] Backup sync options for image and video only when on specific wifi ssid

### DISCUSSION THREAD

**Discussion by ryanwinter** [immich-app/immich#13903] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I, and alot of other people use cloudflare to proxy the immich gallery. The issue I see is that videos just keep failing to upload because of a 100MB limit, however photos are fine.

I would propose additional configuration:
1. Upload photos only when connected to specified wifi SSID
1. Upload videos only when connected to specified Wifi SSID

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4176

**Comment by ryanwinter** [comment]
Slightly different discussion, but I guess they are overlapped with the solution. I'll post comments there.
